src/uu/cp/src/cp.rs

```diff
@@ -769,6 +774,22 @@ impl Options {
         // if not executed first.
         preserve_attributes.sort_unstable();
 
+        // Parse attributes to require preserve
+        let require_preserve_attributes: Vec<Attribute> = if reduced_preserve_requirements_case {
+            preserve_attributes
+                .clone()
+                .into_iter()
+                .filter(|attribute| match attribute {
+                    #[cfg(feature = "feat_selinux")]
+                    Attribute::Context => false,
+                    Attribute::Xattr => false,
```

Comment:
What about the case of `--preserve=all,context`? Should it still not require preserving the context?